conans/client/cmd/export_linter.py

```diff
@@ -76,6 +76,8 @@ def _accept_message(msg):
             return False
         if symbol in ("bare-except", "broad-except"):  # No exception type(s) specified
             return False
+        if symbol == "import-error" and msg.get("line") > 5:  # Import of a conan python package
```

Comment:
Some imports might be legit besides line 5.
Why not checking the column, to see if it is indented?